index.html

```diff
@@ -475,6 +475,14 @@ <h3>Ecosystem Overview</h3>
 information as a valid document.
         </p>
 
+        <p class="note" title="Human-readable text in English is illustrative">
```

Comment:
```suggestion
        <p class="note" title="Human-readable texts in English are illustrative">
```

The note's text refers to plural, the title should, I believe, do the same...